app/src/main/java/nl/rijksoverheid/en/applifecycle/AppLifecycleViewModel.kt

```diff
@@ -16,10 +17,11 @@ import nl.rijksoverheid.en.config.AppConfigManager
 import nl.rijksoverheid.en.config.saveIsTestPhaseVersion
 import nl.rijksoverheid.en.lifecyle.Event
 
+@SuppressLint("StaticFieldLeak")
 class AppLifecycleViewModel(
     private val appLifecycleManager: AppLifecycleManager,
     private val appConfigManager: AppConfigManager,
-    private val context: Context
+    private val applicationContext: Context
```

Comment:
I don't really think we need to rename this to `applicationContext`, it's still a context and we are passing the application context in the right places, although that's an implementation detail really.

Reply:
I changed the name only it to indicate the scope of this context object (lint complained about leaking the context here).
Just for readability. It's easily reverted if you want that :)